cli/CHANGELOG.md

```diff
@@ -1,25 +1,23 @@
  <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
 ## 12.5.0
 
-_Released 01/31/2023 (PENDING)_
+_Released 01/31/2023_
 
 **Features:**
 
-- Easily debug failed CI test runs right from your local Cypress app with the new Debug page, powered by Cypress Cloud.  For more details, see the [Debug documentation](https://on.cypress.io/debug-page). Addressed in [#25488](https://github.com/cypress-io/cypress/pull/25488).
+- Easily debug failed CI test runs recorded to the Cypress Cloud from your local Cypress app with the new Debug page. Please leave any feedback [here](https://github.com/cypress-io/cypress/discussions/25649). Your feedback will help us make decisions to improve the Debug experience. For more details, see [our blog post](https://on.cypress.io/debug-page-release). Addressed in [#25488](https://github.com/cypress-io/cypress/pull/25488).
 
-**Bugfixes:**
+**Performance:**
 
-- Fixed an issue where alternative Microsoft Edge Beta, Canary, and Dev binary versions were not being discovered by Cypress. 
-  Fixes [#25455](https://github.com/cypress-io/cypress/issues/25455).
+- Improved memory consumption in `run` mode by removing reporter logs for successful tests. Fixes [#25230](https://github.com/cypress-io/cypress/issues/25230).
 
-**Performance:**
+**Bugfixes:**
 
-- Improved memory consumption in `run` mode by removing reporter logs for successful tests. 
-  Fixes [#25230](https://github.com/cypress-io/cypress/issues/25230).
+- Fixed an issue where alternative Microsoft Edge Beta, Canary, and Dev binary versions were not being discovered by Cypress. Fixes [#25455](https://github.com/cypress-io/cypress/issues/25455).
 
-**Dependency Updates:**
+<!-- **Dependency Updates:** // this was committed with 'fix' instead of 'dependency' -->
```

Comment:
Since dependabot generated those commits, do we need to change the messaging moving forward? Or does this change just get lumped under buxfixes?
```suggestion
**Dependency Updates:**
```

Reply:
The dependabot PR title needs to be updated to use `dependency: xyz`. I assume we configured it initially so I just need to track it down.